sdk-testing/src/test/java/io/dapr/actors/client/ActorClientStubTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) Microsoft Corporation and Dapr Contributors.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.client;
+
+import io.dapr.actors.ActorId;
+import io.dapr.actors.ActorType;
+import org.junit.Test;
+import org.mockito.Mockito;
+import reactor.core.publisher.Mono;
+
+import java.nio.charset.StandardCharsets;
+
+import static org.junit.Assert.assertEquals;
+
+public class ActorClientStubTest {
+
+  @ActorType(name = "MyActor")
+  public interface MyActor {
+
+    void setName(String name);
+
+    String getName();
+  }
+
+  /**
+   * Example using ActorClientStub to have a mock implementation of an Actor invocation.
+   */
+  @Test
+  public void testMocking() {
+    // Changing this test can mean a breaking change since apps might mock based on some internal behavior of the SDK.
+    ActorClientStub mock = Mockito.mock(ActorClientStub.class);
+    Mockito.when(
+        mock.invoke("MyActor", "100", "getName", null))
+        .thenReturn(Mono.just("\"mock result 100\"".getBytes(StandardCharsets.UTF_8)));
+    Mockito.when(
+            mock.invoke("MyActor", "100", "setName", "\"My Name\"".getBytes(StandardCharsets.UTF_8)))
+        .thenReturn(Mono.just("".getBytes(StandardCharsets.UTF_8)));
+
+    ActorProxyBuilder<MyActor> builder = new ActorProxyBuilder(MyActor.class, mock);
```

Comment:
Passing a diamond operator makes it clear for the compiler that you're using the generic type. You then leave it to the compiler to infer that generic type.

```suggestion
    ActorProxyBuilder<MyActor> builder = new ActorProxyBuilder<>(MyActor.class, mock);
```